packages/x-components/src/components/snippet-callbacks.vue

```diff
@@ -0,0 +1,86 @@
+<template>
```

Comment:
You can extend from NoElement instead of Vue and remove this tag

```html
<template>
  <GlobalXBus v-on="eventListeners" />
</template>
```

`export default class SnippetCallbacks extends NoElement`

Reply:
You shoudn't even need the `NoElement` at all. The `NoElement` should be used when you want to render an slot first child, or when you want to render nothing (and you don't want to write an empty render function). But in this case you are rendering a `GlobalXBus` so it has no place here.